xclim/testing/tests/test_units.py

```diff
@@ -28,6 +28,7 @@ def test_hydro(self):
         with units.context("hydro"):
             q = 1 * units.kg / units.m**2 / units.s
             assert q.to("mm/day") == q.to("mm/d")
+            assert q.to("mm/day").magnitude == 24 * 60**2
```

Comment:
Did you mean to put: 
```suggestion
            assert q.to("mmday").magnitude == 24 * 60**2
```
?

Reply:
ouaip ! Merci, un peu zoné aujourd'hui...